framework/wazuh/cluster/dapi/dapi.py

```diff
@@ -108,9 +108,12 @@ def check_wazuh_status(self):
             return
         basic_services = ('wazuh-modulesd', 'ossec-remoted', 'ossec-analysisd', 'ossec-execd', 'wazuh-db')
         status = operator.itemgetter(*basic_services)(manager.status())
+        status_dict = {}
+        for i in range(0,len(basic_services)):

```

Comment:
It does not make sense at all. I don't see the point creating a list from a dict and then creating a dict from the list again. I would have done the following:
```python
status = manager.status()
required_status = {k: status[k] for k in basic_services}
```